episodes/pathology.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Working with Pathology Images"
+teaching: 60
+exercises: 30
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- What are some open libraries can be used with pathology?
+- How are pathology slides different from a photo?
+- What is the structure of a pathology slide?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Describe some open source tools and libraries for pathology images
+- Discuss  common file formats for pathology slides
+- Explain tiling and pyramidal file formats
+- Show basic code in openslide
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Introduction
+
+Pathologists deal with multiple types of images. Some pathologists may use photography when recording gross (or macroscopic) pathology data. The processing of digital photography is a straightforward application of general image processing. However the workhorse type image of pathology is the histopathological image, and this is very different from a photograph. 
+
+In the below picture you can see an image with gross and histopathology of signet ring cell carcinoma metastasis to the ovary. The histopathology images are stained microscopy image. The gross pathology images are photographs.
```

Comment:
maybe it could be explained what signet ring cell carcinoma means...

Reply:
There is a reason I picked this cancer, but we want to not get too side tracked into medical stuff. But I will definitely I will specify which are photos and which are histo images. I forget that this is not obvious.